### ISSUE TITLE
Unable to finish downloading

### ISSUE THREAD

**Issue by agaldran** [jleuschn/dival#41]
Hello!

Thanks for this amazing work, it looks like tons of care have been put in building this library.

Unfortunately, it seems I am unable to even use the datasets functionality. I have tried twice the following:
```
from dival import get_standard_dataset
dataset = get_standard_dataset('lodopab')
```
And in every case it manages to download the first six files, but when it gets to downloading file 7/9: 'patient_ids_rand_test.csv', it crashes with a key error:
![image](https://github.com/jleuschn/dival/assets/4498444/b5914b92-69c5-4a45-8ef1-ee747d9eba15)

BTW I also tried with:
```
import dival
dival.datasets.lodopab_dataset.download_lodopab()
```
And got almost the same error:
![image](https://github.com/jleuschn/dival/assets/4498444/6ed411c2-a535-4399-a2a0-3579c5880e00)

Any clues about how to fix this? Thanks!

**Comment by agaldran** [comment]
Hello again,

I manage to make this work by downloading manually those three csv files from zenodo and putting them in the corresponding directory, but I am leaving this issue open in case you want to fix it? Otherwise, please feel free to close it.

Cheers,

Adrian

**Comment by jleuschn** [comment]
Hey Adrian,

seems to me that at least by now not only the csv but also the zip files couldn't be downloaded anymore.
Both should now be fixed after #42 (thanks to @julianbopp).

Best,
Johannes
